CHANGELOG.md

```diff
@@ -27,6 +28,13 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Fixes and removes bad console logs from the `Pagination` and `List` components.
 - Fixes auto-generated `id`s in the `Tabs` component.
 
+### Updates
+
+- Updates the following packages: multiple `@storybook/...` packages, `autoprefixer`, `eslint-plugin-storybook`, `husky`, `lint-staged`, `semantic-release-slack-bot`, `storybook-addon-designs`.
+- Removed the following packages: `postcss`, `pretty-quick`.
+- Added `@chakra-ui/react` and `@chakra-ui/system` as "peerDependencies".
+- Updates the node version in Github Action workflows from Node v12 to Node v14.
```

Comment:
What is the node version for other NYPL React apps?

Reply:
It varies but I think we're trying to make node v14 the minimum and moving onto 16+ in the future. Some apps on AWS are still on node v10 and IT has been moving them to v12 (as a minimum requirement per AWS). I'm also wary of jumping up to v16 or higher from v12 so the slower progression _seems_ safer to me.

Reply:
The Library Card App I'm working on on the side is on node 10 on AWS. We can't move to node 12 because it causes memory issues... so some apps are _still_ on older versions. But this doesn't affect how the DS is worked on, deployed, or consumed.